src/components/Home/Board/index.tsx

```diff
@@ -414,6 +415,26 @@ const products: Product[] = [
             },
         },
     },
+    {
+        name: 'MCP Server',
+        color: 'seagreen',
+        Icon: IconCode,
+        description: 'Give your agents full control over PostHog.',
+        types: ['AI'],
+        features: [
+            { title: 'Use PostHog in your editor', Icon: IconCode },
+            { title: 'Automate tasks based on PostHog data', Icon: IconSparkles },
+            { title: 'Build agent workflows with PostHog', Icon: IconBolt },
+        ],
```

Comment:
All of the products have images, so it would be nice for this one to have an image too. 
```suggestion
        ],
        Images: () => {
            return (
                <>
                    <div className="block">
                        <CloudinaryImage src="https://res.cloudinary.com/dmukukwp6/image/upload/robot_6d2cab1b66" />
                    </div>
                </>
            )
        },
```